references/classification/presets.py

```diff
@@ -18,10 +18,11 @@ def __init__(
         random_erase_prob=0.0,
         center_crop=False,
     ):
-        trans = (
-            [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
+        trans = [transforms.ToImageTensor()]
+        trans.append(
+            transforms.RandomResizedCrop(crop_size, interpolation=interpolation, antialias=True)
             if center_crop
-            else [transforms.CenterCrop(crop_size)]
+            else transforms.CenterCrop(crop_size)
         )
```

Comment:
Adapted cherrypick from #6642

Reply:
Looks like the `center_crop` was unused and buggy. We'll need to remove it later.

Reply:
The issue is now fixed on main. I've cleaned up the code and merged it here. Unfortunately due to this bug, I had to restart the runs on the Classification models, so it's going to take longer to verify that the accuracies match.